src/agentscope/file_manager.py

```diff
@@ -69,6 +74,11 @@ def dir_invoke(self) -> str:
         """The directory for saving api invocations."""
         return self._get_and_create_subdir(_DEFAULT_SUBDIR_INVOKE)
 
+    @property
+    def file_db(self) -> str:
```

Comment:
Maybe unified into `dir_db` or `path_db`? The prefix `file_` is a little strange.